build_config/lib/src/build_config.dart

```diff
@@ -74,7 +74,7 @@ class BuildConfig {
   /// Returns a parsed [BuildConfig] file in [path], if one exists, otherwise a
   /// default config.
   ///
-  /// [path] maybe be a directory which contains a `build.yaml`.
+  /// [path] shouldd the path to a directory which may contain a `build.yaml`.
```

Comment:
should be

Reply:
You'd think typing wouldn't be the hard part...